src/applications/search/containers/SearchApp.jsx

```diff
@@ -54,16 +46,24 @@ class SearchApp extends React.Component {
       currentResultsQuery: userInputFromURL,
       page: pageFromURL,
       typeaheadUsed,
+      formWasSubmitted: false,
+      landedWithSearchInput: false,
     };
+
+    this.searchFieldRef = createRef(null);
```

Comment:
Latest changes: added a ref to assign to the web component search input for focusing.